### PR TITLE
Serilog implementation

### PR DESCRIPTION
**Pull request by rgr182**

### CODE REVIEW

File: ScraperAdmin/ScraperAdmin.csproj.user
Comment (Paroxyse) [inline ScraperAdmin/ScraperAdmin.csproj.user:1]: This file shouldn't be in version control 
image

File: ScrapperCron/Program.cs [hunk 1]
```diff
@@ -1,49 +1,27 @@
...
-using Microsoft.Extensions.Hosting;
-using ScrapperCron.Services;
-using System;
-using System.Threading.Tasks;
+﻿using Microsoft.Extensions.Hosting;
+using ScrapperCron.Infrastructure;
+using Serilog;
 
-namespace Cron_BolsaDeTrabajo
+namespace ScrapperCron
```
Comment (Paroxyse) [inline ScrapperCron/Program.cs]: Typo, a scrapper and a scraper are not the same thing

**Review by Paroxyse — CHANGES_REQUESTED** [review]
Remove ScraperAdmin/ScraperAdmin.csproj.user from version control

File: ScraperAdmin/ScraperAdmin.csproj.user
Comment (rgr182) [reply ScraperAdmin/ScraperAdmin.csproj.user:1]: ev
 that file is ignored by git ignore rightnow i can not see it in the git repository

Comment (rgr182) [reply ScraperAdmin/ScraperAdmin.csproj.user:1]: Done!

File: ScrapperCron/Program.cs (hunk 1 above)
Comment (rgr182) [reply ScrapperCron/Program.cs]: Done!

**Comment by rgr182** [comment]
Closed Meanwhile merge issues get solved

**Pull request closed without merging**